library/crates/Cargo.toml

```diff
@@ -7,18 +7,20 @@ path = "" # ignored by cargo raze
 
```

Comment:
Summary of changes:

- Sorted crate dependencies alphabetically
- Added `derivative`, `enum_dispatch` and `uuid`, used by `typedb-client`
- Added `executor` and `thread-pool` features to `futures`, used by `typedb-client`
- Added `tokio`, used by `typedb-client` and `factory`
- Added `grpc-protobuf` and `protoc-rust-grpc`, used by `typedb-protocol`
- Updated `grpc`, `grpc-compiler`, `protobuf` and `protobuf-codegen`, used by `typedb-client` and `typedb-protocol`
- Removed `tls-api` and `tls-api-stub`, no longer in use (they were used by `typedb-client`)